lang/LangSource/PyrParseNode.cpp

```diff
@@ -1375,6 +1375,9 @@ void PyrMethodNode::compile(PyrSlot* result) {
         */
     } else if (slotRawSymbol(&gCompilingMethod->name) == s_doesNotUnderstand) {
         methType = methNormal;
+    } else if (slotRawSymbol(&gCompilingMethod->name) == s_doesNotUnderstandWithKeys) {
+        // this branch could be incorrect as I don't know what it does.
```

Comment:
is this something to do before we finish up?

Reply:
Well it hasn't caused any bugs? 

... but I have no idea what that line does because I couldn't figured out what an 'abnormal' method would be.
I just kept the symmetry.

Reply:
I see. The comment is misleading because you added the code yourself :)

Maybe you could explicitly say that this code was copied from `s_doesNotUnderstand` without knowing what it does?

Reply:
Actually, I've removed the comment now as the options are
```
enum {
    methNormal = 0,
    methReturnSelf,
    methReturnLiteral,
    methReturnArg,
    methReturnInstVar,
    methAssignInstVar,
    methReturnClassVar,
    methAssignClassVar,
    methRedirect,
    methRedirectSuper,
    methForwardInstVar,
    methForwardClassVar,
    methPrimitive,
    methBlock
};
```
It is clearly a methNormal.